event.go

```diff
@@ -0,0 +1,28 @@
+package xendit
+
+type EventInvoicePaid struct {
+	ID                 string               `json:"id"`
+	Fees               []InvoiceFee         `json:"fees"`
+	Amount             int                  `json:"amount"`
+	Status             string               `json:"status"`
+	Created            string               `json:"created"`
+	IsHigh             bool                 `json:"is_high"`
+	PaidAt             string               `json:"paid_at"`
+	Updated            string               `json:"updated"`
+	UserID             string               `json:"user_id"`
+	Currency           string               `json:"currency"`
+	PaymentID          string               `json:"payment_id"`
+	ExternalID         string               `json:"external_id"`
+	PaidAmount         int                  `json:"paid_amount"`
+	MerchantName       string               `json:"merchant_name"`
+	PaymentMethod      string               `json:"payment_method"`
+	PaymentChannel     string               `json:"payment_channel"`
+	PaymentDetails     InvoicePaymentDetail `json:"payment_details"`
```

Comment:
Could you declare this as a pointer since this field is optional? `*InvoicePaymentDetail`

Reply:
sure! 
i also fixed it here 930783ebea4fb46db32743de3ca951eeaa58c44b